cli/types/index.d.ts

```diff
@@ -866,6 +866,12 @@ declare namespace Cypress {
      */
     hash(options?: Partial<Loggable & Timeoutable>): Chainable<string>
 
+    /**
+     * Invoke a function in an array of functions.
+     * @see https://on.cypress.io/invoke
+     */
+    invoke<T extends (...args: any[]) => any, Subject extends T[]>(index: number): Chainable<ReturnType<T>>
+

```

Comment:
wish here I could write `<ReturnType<Subject[index]>>` to point at specific function